components/eamxx/cime_config/config_compsets.xml

```diff
@@ -81,6 +81,13 @@
       <support_level>Experimental, under development</support_level>
   </compset>
 
+  <compset>
+      <alias>F2010-SCREAMv1-X6T</alias> <!-- Coupled land-atm compset (using SCREAMv1), with X6T topography-->
+      <lname>2010_SCREAM%X6T_ELM%SPBC_CICE%PRES_DOCN%DOM_SROF_SGLC_SWAV_SIAC_SESP</lname>
```

Comment:
I saw a lot of repetitive keys here from the screamv1, so I copied. Is this correct to do?